src/components/ErrorBoundary.js

```diff
@@ -28,16 +26,8 @@ class ErrorBoundary extends React.Component {
     if (!this.state.hasError) {
       return this.props.children;
     }
-
     return (
-      <div className={getBEMClassName('alert', ['error'])}>
-        <span className={getBEMClassName('alert__icon', ['wide'])}>
-          <FAIcon icon="exclamation-circle" />
-        </span>
-        <Body>
-          Er ging helaas iets fout!
-        </Body>
-      </div>
+      <ErrorMessage>Er ging helaas iets fout!</ErrorMessage>
```

Comment:
Should maybe this also be a translatable string?

Reply:
yeah, but so far I've only been adding it for new code because the `id`s everywhere are bothering me. I've got another ticket to set up the actual i18n toolchain with message extraction similar to the open-forms repo.